bin/stacks/api-stack.ts

```diff
@@ -453,9 +453,9 @@ export class APIStack extends cdk.Stack {
       metric: api.metricServerError({
         period: Duration.minutes(5),
         // For this metric 'avg' represents error rate.
-        statistic: 'avg',
+        statistic: 'sum',
```

Comment:
what does sum mean in this context?

Reply:
It takes the absolute count over the time period, rather than the avg. It makes the spikes more pronounced and easier to alert on. There shouldn't be any reason we have 500 errors so even small amounts are worth looking into.

Reply:
makes sense!